stdlib/public/core/Sequence.swift

```diff
@@ -1412,6 +1412,35 @@ extension Sequence {
 }
 
 extension Sequence {
+
+  @inline(__always)
+  public
+  func _copyContents( buffer: UnsafeMutableBufferPointer<Element>,
```

Comment:
Here and elsewhere, can you make sure to follow prevailing stdlib convention, specifically:
* Place access modifiers on the same line as `func`
* Use no space after the opening parenthesis
* Use no space before colon and one space after colon (unless stating conformances or inheritance, in which case use one space before and after)
* Use two-space tabs and 80-character columns
* Place braces on the same line as `else`, `for`, etc. (including on the same line as the function return type); place `else` on the same line as `guard`
* Use no spaces before and after `..<`